org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -937,6 +969,42 @@ SsoSynapseLlmProdAdmin:
     principalId: !Ref SynapseLlmProdAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
 
+SsoSynapseLlmProdDeveloper:
```

Comment:
This is the developers group in the SynapseLLM account (uses the generic developers permission set).